src/Cmfd.cpp

```diff
@@ -368,21 +345,21 @@ void Cmfd::computeDs(int moc_iteration) {
 
           /* Get diffusivity and flux for Mesh cell */
           d = _materials[cell]->getDifCoef()[e];
-          flux = _old_flux[cell*_num_cmfd_groups+e];
+          flux = _old_flux->getValue(cell, e);
           cell_next = getCellNext(cell, surface);
 
           /* Set halfspace sense of the Surface */
-          if (surface == 0 || surface == 1)
+          if (surface == SURFACE_X_MIN || surface == SURFACE_Y_MIN)
             sense = -1.0;
           else
             sense = 1.0;
 
           /* Set the length of this Surface and the perpendicular Surface */
-          if (surface == 0 || surface== 2) {
+          if (surface == SURFACE_X_MIN || surface== SURFACE_X_MAX) {
```

Comment:
I don't see the point of this ``else-if`` block. It looks like irregardless of the value of ``surface``, the ``length`` and ``length_perpen`` variables will be set to the same value. Am I missing something here @samuelshaner?